rts/Lua/LuaUnsyncedCtrl.cpp

```diff
@@ -3382,6 +3384,226 @@ int LuaUnsyncedCtrl::SetBuildFacing(lua_State* L)
 }
 
 
+static bool ParseBuildInfo(
+	lua_State* L,
+	BuildInfo& buildInfo,
+	const char* caller,
+	int idx
+) {
+	for (int i=0; i<5; i++) {
+		if (!lua_isnumber(L, idx+i)) {
+			luaL_error(L, "%s(): bad build info type", caller);
+			return false;
+		}
+	}
+
+	int unitDefID = lua_toint(L, idx);
+	float3 pos = {lua_tofloat(L, idx+1), lua_tofloat(L, idx+2), lua_tofloat(L, idx+3)};
+	int facing = lua_toint(L, idx+4);
```

Comment:
IIRC there's some sort of facing parser in LuaUtils that makes facing args work both with numbers and things like "south", might want to optionally check it out

Reply:
No, all functions using facing (including `SetBuildFacing`) just parse an int.